src/renuka/Shop.java

```diff
@@ -0,0 +1,98 @@
+package renuka;
+
+public class Shop {
+		
+	 int maggieQ;
+	 int dosaQ;
+	 int pouchesQ;
+	 int panipuriQ;
+	 int masalaQ;
+	
+	
+	void setQ(int reqMaggieQ, int reqDosaQ, int reqPouchesQ, int reqPanipuriQ, int reqMasalaQ) {
+		maggieQ = reqMaggieQ;
+		dosaQ = reqDosaQ;
+		pouchesQ = reqPouchesQ;
+		panipuriQ = reqPanipuriQ;
+		masalaQ = reqMasalaQ;
+	}	
+	
+	void requestedQ(int reqMaggieQ, int reqDosaQ, int reqPouchesQ, int reqPanipuriQ, int reqMasalaQ) {
+		if(reqMaggieQ <= maggieQ) {
+		maggieQ = maggieQ - reqMaggieQ;
+		System.out.println("Remaining Qty is: " +maggieQ);
+		}
+		else {
+			System.out.println("Insufficient Quantity");
+			maggieQ = 0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct